app/scripts/controllers/visualizer.js

```diff
@@ -4,25 +4,15 @@
 angular.module('openhimConsoleApp')
   .controller('VisualizerCtrl', function ($scope, $http, $interval, $window, login, Api, Alerting, Fullscreen) {
 
+    // initialize global variables
+    var settingsStore = {}; // a place the push current settings when switching to fullscreen
+    var visualizerUpdateInterval, updatePeriod, diffTime, lastUpdate, maxSpeed, pad;
+
     $scope.loadingVisualizer = true;
     $scope.loadingVisualizerError = false;
     $scope.loadingVisualizerErrorMsgs = [];
```

Comment:
The above two don't seem to be used anymore, but maybe they are kept in for future use?